android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,18 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="tech.bam.RNBraintreeDropIn">
 
+    <application>
```

Comment:
It would be better for users to make this a configuration step for using the library
Not everybody wants to have this in their code
See example of 3D Secure https://github.com/bamlab/react-native-braintree-payments-drop-in#3d-secure